### ISSUE TITLE
SearchBox and Live Persona Card not working.

### ISSUE THREAD

**Issue by sudeshkumara** [YannickRe/spfx-msgraph-peoplesearch#9]
@YannickRe  Thank you for this cool web part and sorry to ask this under issues.

I've implemented this in our tenant and it's working fine except for the SearchBox and Live Persona Card.

![image](https://user-images.githubusercontent.com/44965299/101760014-53637d80-3b00-11eb-9d88-66471daa634d.png)

![image](https://user-images.githubusercontent.com/44965299/101759034-1945ac00-3aff-11eb-9874-7de86fb8aeac.png)

static search also working, how can I add a search box? could you please show me the direction for that?

When it's come to Live persona card, it's working on the SharePoint default People web part and I've enabled the feature in configurations for peoplesearch web part. But it's not working.
![image](https://user-images.githubusercontent.com/44965299/101759482-abe64b00-3aff-11eb-9434-24f8340cddde.png)
![image](https://user-images.githubusercontent.com/44965299/101759794-10090f00-3b00-11eb-987f-56925ebd4fde.png)

**Comment by YannickRe** [comment]
This is strange behavior, and shouldn't really occur. I'm not seeing any issues in my deployed instances.
Things to try:
- Do you have the latest version from GitHub?
- Can you check for javascript errors in the console? This might explain both problems
- Select "none", save and fully reload the page. Select "Searchbox" again. If possible, can you screencapture this? It'll be easier for me to understand what is going on.

**Comment by sudeshkumara** [comment]
@YannickRe 

Sorry for the late reply,

But I've resolved the issues.
**Search Box Issue** : Actually there are no issues with the search box. What happened was that the search box was hidden because of custom CSS.
**Persona Card Issue** : I added the following parameters to the people search query.  (id,displayName,mail,jobTitle)

![image](https://user-images.githubusercontent.com/44965299/102010373-ad17b200-3d63-11eb-9d47-a8b8d774c92a.png)

It did not show any query errors. But somehow when I removed all these parameters the personal card worked fine.
I just wanted to hide the phone number and achieved this by editing the name Field. **Manage persona fields**

Thank you very much @YannickRe  for this package.
